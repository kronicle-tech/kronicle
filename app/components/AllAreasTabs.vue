<template>
  <div class="my-3">
    <b-button to="/all-areas">Areas</b-button>
  </div>
</template>

<script lang="ts">
import Vue from 'vue'
import { BButton } from 'bootstrap-vue'

export default Vue.extend({
  components: {
    'b-button': BButton,
  },
})
</script>
