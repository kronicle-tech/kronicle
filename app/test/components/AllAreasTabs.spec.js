import { mount } from '@vue/test-utils'
import AllAreasTabs from '@/components/AllAreasTabs.vue'

describe('AllAreasTabs', () => {
  let propsData
  let wrapper
  const createWrapper = () => {
    wrapper = mount(AllAreasTabs, { propsData })
  }

  beforeEach(() => {
    propsData = {}
  })

  afterEach(() => {
    wrapper.destroy()
    wrapper = null
  })

  test('renders the component', () => {
    createWrapper()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
