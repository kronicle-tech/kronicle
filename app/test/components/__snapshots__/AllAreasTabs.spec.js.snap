// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllAreasTabs renders the component 1`] = `"<div class=\\"my-3\\"><a href=\\"/all-areas\\" target=\\"_self\\" class=\\"btn btn-secondary\\">Areas</a></div>"`;
